### PR TITLE
jtac radio key and weyland-yutani radio key

### LINKED ISSUE #1437: 5P JTAC Radio Encryption Key

### LINKED ISSUE #1508: 3P JTAC Radio Encryption Key

### LINKED ISSUE #1619: JTAC Radio Encryption Key

### LINKED ISSUE #1678: 3P JTAC Radio Encryption Key

### LINKED ISSUE #1790: 3P JTAC Radio Encryption Key

### LINKED ISSUE #1868: 5P JTAC Radio Encryption Key

### LINKED ISSUE #1937: 5P JTAC Radio Encryption Key

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR

Fixes #1937, Fixes #1868, Fixes #1790, Fixes #1678, Fixes #1619, Fixes #1508, Fixes #1437

**Changelog**

:cl: DangerRevolution
- add: Added JTAC Radio Key
- add: Add Weyland-Yutani radio key

### CODE REVIEW

**Comment by DangerRevolution** [comment]
just as a side comment, it looks like CM13 removed the Tactics Key, can't find much mention of it on the github aside from changelog entries.

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml [hunk 1]
```diff
@@ -697,9 +697,9 @@
       #- id: CMTacticsRadioEncryptionKey TODO: Add Tactics Key
       #  amount: 3
       #  points: 3
-      #- id: CMJTACRadioEncryptionKey TODO: Add JTAC Key.
-      #  amount: 3
-      #  points: 3
+      - id: CMJTACRadioEncryptionKey
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml:700]:
```suggestion
      - id: CMEncryptionKeyJTAC
```

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml [hunk 2]
```diff
@@ -918,9 +918,9 @@
       #- id: CMTacticsRadioEncryptionKey TODO: Add Tactics Key.
       #  amount: 5
       #  points: 5
-      #- id: CMJTACRadioEncryptionKey TODO: Add JTAC Key.
-      #  amount: 5
-      #  points: 5
+      - id: CMJTACRadioEncryptionKey
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml:921]:
```suggestion
      - id: CMEncryptionKeyJTAC
```

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml [hunk 3]
```diff
@@ -1076,9 +1076,9 @@
       #- id: CMTacticsRadioEncryptionKey TODO: Add Tactics Key
       #  amount: 3
       #  points: 3
-      #- id: CMJTACRadioEncryptionKey TODO: Add JTAC Key
-      #  amount: 3
-      #  points: 3
+      - id: CMJTACRadioEncryptionKey
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml:1079]:
```suggestion
      - id: CMEncryptionKeyJTAC
```

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml [hunk 4]
```diff
@@ -1277,9 +1277,9 @@
       #- id: CMTacticsRadioEncryptionKey TODO: Add Tactics Key.
       #  amount: 3
       #  points: 3
-      #- id: CMJTACRadioEncryptionKey TODO: Add JTAC Key.
-      #   amount: 3
-      #   points: 3
+      - id: CMJTACRadioEncryptionKey
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml:1280]:
```suggestion
      - id: CMEncryptionKeyJTAC
```

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml [hunk 5]
```diff
@@ -1492,9 +1492,9 @@
       - id: CMEncryptionKeyIntel
         amount: 5
         points: 5
-      #- id: CMJTACRadioEncryptionKey
-      #  amount: 5
-      #  points: 5
+      - id: CMJTACRadioEncryptionKey
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml:1495]:
```suggestion
      - id: CMEncryptionKeyJTAC
```

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml [hunk 6]
```diff
@@ -1661,9 +1661,9 @@
       #- id: CMTacticsRadioEncryptionKey TODO: Add Tactics Key.
       #  amount: 5
       #  points: 5
-      #- id: CMJTACRadioEncryptionKey TODO: Add JTAC Key.
-      #  amount: 5
-      #  points: 3
+      - id: CMJTACRadioEncryptionKey
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml:1664]:
```suggestion
      - id: CMEncryptionKeyJTAC
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
Wrong ID in a few places is causing tests to fail

**Comment by DangerRevolution** [comment]
> Wrong ID in a few places is causing tests to fail

thank you, must've got confused at some point and was writing the wrong ID lol

**Comment by DangerRevolution** [comment]
test fail doesn't seem related to me

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**